Refresh.GameServer/Database/GameDatabaseContext.Levels.cs

```diff
@@ -391,6 +390,9 @@ public int GetTotalLevelsPublishedByUser(GameUser user, TokenGame game)
                 return null;
         }
     }
+
+    public GameLevel? GetLevelByRootResource(string rootResource) =>
+        this.GameLevels.FirstOrDefault(level => level.RootResource == rootResource);
```

Comment:
Obviously out of scope but we should provide an API endpoint for this :)

`/api/v3/levels/hash/{hash}`